README.md

```diff
@@ -167,6 +168,41 @@ First, [fork](https://guides.github.com/activities/forking/) this repository, in
 When you make a PR to edit an existing package, `dt-bot` should @-mention previous authors.
 If it doesn't, you can do so yourself in the comment associated with the PR.
 
+#### Editing tests on an existing package
+
+There should be a `[modulename]-tests.ts` file, and then any `*.ts` which it imports are considered your test files.
+These files are used to validate the API exported from the `*.d.ts` files which are shipped as `@types/yourmodule`.
+
+Changes to the `*.d.ts` files should include a corresponding `*.ts` file change which shows the API being used, so that someone doesn't accidentally break code you depend on.
+If you don't see any test files in the module's folder, create a `[modulename]-tests.ts`
+
+For example, this change to a function in a `.d.ts` file adding an option
+
+`index.d.ts`:
+
+```diff
+- export function twoslash(body: string): string
++ export function twoslash(body: string, config?: { version: string }): string
+```
+
+`index-tests.ts`:
+```diff
+import {twoslash} from "./"
+
+// $ExpectType string
+const result = twoslash("//")
+
++ // Handle options param
++ const resultWithOptions = twoslash("//", { version: "3.7" })
++ // When the param is incorrect
++ // $ExpectError
++ const resultWithOptions = twoslash("//", {  })
+```
+
+You can validate your changes with either:
+
+ - `npm test` from the root of this repo, which takes changed files into account
+ - `npx tsc` from the modules folder
```

Comment:
```suggestion
 - `npx tsc` from the module's folder
```

although it's a little hard to recommend `tsc` right after showing off `$ExpectError` since `tsc` doesn't understand that comment.